### PR TITLE
chore: remove all aliases other than src = @

### PR DESCRIPTION
**Pull request by SHolleworth**

### CODE REVIEW

File: packages/client/src/services/trades/__mocks__/creditTrades.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { Direction } from "generated/TradingGateway"
-import { ACCEPTED_QUOTE_STATE } from "generated/TradingGateway"
+import { Direction } from "@/generated/TradingGateway"
```
Comment (algreasley) [inline packages/client/src/services/trades/__mocks__/creditTrades.ts:1]: .. squash into one

File: packages/client/src/services/trades/__tests__/trades.test.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
-import { Direction, RfqState } from "generated/TradingGateway"
+import { BehaviorSubject } from "rxjs"
+
+import { Direction, RfqState } from "@/generated/TradingGateway"
```
Comment (algreasley) [inline packages/client/src/services/trades/__tests__/trades.test.ts:3]: squash these too

File: packages/client/src/services/trades/__tests__/trades.test.ts [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
   ACCEPTED_QUOTE_STATE,
   REJECTED_WITH_PRICE_QUOTE_STATE,
-} from "generated/TradingGateway"
-import { BehaviorSubject } from "rxjs"
+} from "@/generated/TradingGateway"
 
 import * as creditService from "../../credit"
 import { RfqDetails } from "../../credit"
-// import * as creditService from "services/credit"
+// import * as creditService from "@/services/credit"
```
Comment (algreasley) [inline packages/client/src/services/trades/__tests__/trades.test.ts:11]: remove comment plz

File: packages/client/src/services/trades/trades.ts [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import { bind } from "@react-rxjs/core"
-import { BlotterService } from "generated/TradingGateway"
+import { map, scan } from "rxjs/operators"
+
+import { BlotterService } from "@/generated/TradingGateway"
```
Comment (algreasley) [inline packages/client/src/services/trades/trades.ts:4]: squash ..

File: packages/client/src/workspace/home/notifications.ts [hunk 5]
```diff
@@ -1,13 +1,14 @@
...
+
 import {
   handleHighlightFxBlotterAction,
   handleHighlightRfqAction,
   TASK_HIGHLIGHT_CREDIT_RFQ,
   TASK_HIGHLIGHT_FX_TRADE,
-} from "client/notifications.openfin"
-import { RT_PLATFORM_UUID_PREFIX } from "client/OpenFin/utils/window"
-import { NotificationActionEvent } from "openfin-notifications"
+} from "@/client/notifications.openfin"
```
Comment (algreasley) [inline packages/client/src/workspace/home/notifications.ts:10]: looks/feels weird .. something for later

File: packages/client/src/client/App/Admin/Admin.tsx [hunk 6]
```diff
@@ -1,11 +1,12 @@
 import { bind } from "@react-rxjs/core"
 import { createSignal } from "@react-rxjs/utils"
-import { withSubscriber } from "client/utils/withSubscriber"
-import { ThroughputAdminService } from "generated/TradingGateway"
 import { concat, merge, of } from "rxjs"
 import { catchError, debounceTime, exhaustMap, map } from "rxjs/operators"
 import styled from "styled-components"
 
+import { withSubscriber } from "@/client/utils/withSubscriber"
+import { ThroughputAdminService } from "@/generated/TradingGateway"
```
[the remark this answers is not in the document]
Comment (SHolleworth) [reply packages/client/src/client/App/Admin/Admin.tsx:8]: I think that sounds good, had something similar before on another project

**Review by algreasley — APPROVED** [review]

**Pull request merged**